apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -173,7 +177,7 @@ class ProgressBubble extends React.Component {
     // If we have an href, wrap in an achor tag
     if (href) {
       bubble = (
-        <a href={href} style={{textDecoration: 'none'}}>
+        <a href={href} style={{textDecoration: 'none'}} className="uitest-ProgressBubble">
```

Comment:
Probably not a big deal, but for a component like this that ends up in the DOM lots of times, I'd prefer not to add a classname that's only needed for uitests. Would it be difficult to avoid this?